Update intro sections (#1232)
Rebilly/api-definitions: openapi/description.md

# Introduction
[comment]: <> (x-product-description-placeholder)
The Rebilly API is built on HTTP and is RESTful.
It has predictable resource URLs and returns HTTP response codes to indicate errors.
It also accepts and returns JSON in the HTTP body.
Use your favorite HTTP/REST library in your programming language when using this API,
or use one of the Rebilly SDKs,

> which are available in [PHP](https://github.com/Rebilly/rebilly-php) and [Javascript](https://github.com/Rebilly/rebilly-js-sdk).

Reviewer: ```suggestion
which are available in [PHP](https://github.com/Rebilly/rebilly-php) and [JavaScript](https://github.com/Rebilly/rebilly-js-sdk).
```

# SDKs

Rebilly provides a JavaScript SDK and a PHP SDK to help interact with the Rebilly API.
However, no SDK is required to use the API.

> Rebilly also provides [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/),

Reviewer 2: ```suggestion
Rebilly also provides [FramePay](https://www.rebilly.com/docs/developer-docs/framepay/),
```

a client-side iFrame-based solution, to help create payment tokens while minimizing PCI DSS compliance burdens
and maximizing your customization ability.

> [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/) interacts with the [payment tokens creation operation](https://all-rebilly.redoc.ly/tag/Payment-tokens/operation/PostToken).

Reviewer 2: This link to docs.rebilly.com is outdated. 
```suggestion
[FramePay](https://www.rebilly.com/docs/developer-docs/framepay/) interacts with the [payment tokens creation operation](https://all-rebilly.redoc.ly/tag/Payment-tokens/operation/PostToken).
```

Rebilly provides collections filtering. Use the `?filter` parameter on collections to define which records should be shown in the response.

Format description:

- Fields and values in the filter are separated with `:`: `?filter=firstName:John`.

- Sub-fields are separated with `.`: `?filter=billingAddress.country:US`.

- Multiple filters are separated with `;`: `?filter=firstName:John;lastName:Doe`. \
  They are joined with `AND` logic. Example: `firstName:John` AND `lastName:Doe`.

- To use multiple values, use `,` as values separators: `?filter=firstName:John,Bob`. \
  Multiple values specified for a field are joined with `OR` logic. Example: `firstName:John` OR `firstName:Bob`.

- To negate the filter, use `!`: `?filter=firstName:!John`.

- To negate multiple values, use: `?filter=firstName:!John,!Bob`.
  This filter rule excludes all `Johns` and `Bobs` from the response.

- To use range filters, use: `?filter=amount:1..10`.

- To use a gte (greater than or equals) filter, use: `?filter=amount:1..`.
  This also works for datetime-based fields.
  
- To use a lte (less than or equals) filter, use: `?filter=amount:..10`.
  This also works for datetime-based fields.

> - To create [specified values lists](https://user-api-docs.rebilly.com/#tag/Lists) and use them in filters, use: `?filter=firstName:@yourListName`. \

Reviewer 2: Link to all api instead.

Reviewer 2: ```suggestion
- To create [specified values lists](https://all-rebilly.redoc.ly/#tag/Lists) and use them in filters, use: `?filter=firstName:@yourListName`. \
```
